draft-ietf-quic-transport.md

```diff
@@ -1218,12 +1215,12 @@ ack_delay_exponent (0x0007):
 
 ### Values of Transport Parameters for 0-RTT {#zerortt-parameters}
 
-Transport parameters from the server MUST be remembered by the client for use
-with 0-RTT data.  If the TLS NewSessionTicket message includes the
-quic_transport_parameters extension, then those values are used for the server
-values when establishing a new connection using that ticket.  Otherwise, the
-transport parameters that the server advertises during connection establishment
-are used.
+A client that attempts to send 0-RTT data MUST remember the transport parameters
+used by the server.  The transport parameters that the server advertises during
+connection establishment apply to all connections that are resumed using the
+keying material established during that handshake.  Remembered transport
+parameters apply to the new connection until the handshake completes and new
+transport parameters from the server can be provided.
```

Comment:
So, what is our current approach for ensuring that the transport parameter change does not put client in a sudden violation of them?  Do we have a text somewhere else in the text saying that flow control parameters may only increase, or does this still need to be addressed?

Reply:
The existing text says that if the server accepts 0-RTT it can't change transport parameters to values that the client might violate with its 0-RTT: https://quicwg.github.io/base-drafts/draft-ietf-quic-transport.html#rfc.section.7.4.2